lib/stripe/subscriptions/subscription.ex

```diff
@@ -164,6 +164,7 @@ defmodule Stripe.Subscription do
                optional(:items) => [
                  %{
                    :plan => Stripe.id() | Stripe.Plan.t(),
+                   optional(:id) => Stripe.id() | binary(),
```

Comment:
Is this documented anywhere?

https://stripe.com/docs/api/subscriptions/create